packages/posthog/.env.sample

```
@@ -3,10 +3,12 @@
 # https://app.posthog.com/settings/user-api-keys
 POSTHOG_PERSONAL_API_KEY=
 POSTHOG_PROJECT_ID=
-# Optional — defaults to https://us.posthog.com
-# POSTHOG_API_HOST=
+# Region app host for the management API — no default; a wrong-region call
+# silently finds nothing. The production project is on the EU cloud.
+POSTHOG_API_HOST=https://eu.posthog.com
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The POSTHOG_API_HOST key should go before the POSTHOG_PERSONAL_API_KEY key

(UnorderedKey)